src/resources/MachineLearning/Document.ts

```diff
@@ -0,0 +1 @@
+export type DocumentRequirementStatus = 'OK' | 'INSUFFICIENT_DOCUMENTS' | 'NB_OF_DOCUMENTS_OVER_LIMIT';
```

Comment:
Personally, I would have gone with `'OK' | 'NOT_ENOUGH_DOCUMENTS' | 'TOO_MANY_DOCUMENTS'`, but that would have to be changed at the API level as well so feel free to ignore 😁